Makefile

```diff
@@ -142,8 +142,12 @@ LOCAL_BUILD=true
 .PHONY: dev-image dev-test dev-clean
 ## Build a local version of Calico based on the checked out codebase.
 dev-image: $(addsuffix -dev-image, $(filter-out calico felix, $(RELEASE_REPOS)))
+
+# Dynamically declare new make targets for all calico subprojects...
 $(addsuffix -dev-image,$(RELEASE_REPOS)): %-dev-image: ../%
-	@cd $< && export TAG=$$($(TAG_COMMAND)); make image tag-images \
+	echo "TARGET:"
+	echo $< 
+	@cd $< && export TAG=$$($(TAG_COMMAND)); make image retag-build-images-with-registries ARCHES=amd64 \
```

Comment:
I think `TAG` needs to be `IMAGETAG`

Reply:
Ah ok
Will fix

Reply:
looks like BUILD_IMAGE and IMAGETAG are Both required?

Reply:
yeah , its working now.

Reply:
IMAGETAG =s set at the bottom of the make declaration ......   i think all set now  ? 

seems to pass for me